### ISSUE TITLE
Error for Rss feeds 

### ISSUE THREAD

**Issue by stmolivier** [iCAPLyon1/NotificationBundle#12]
On a V6.3.1, on the notification management page  /icap_notification/parameters
the link RSS feed leads to a error 500 :
I've tried the "regenerate feed url", "modify" button, but the result is the same :
``` 
<error code="500" message="Internal Server Error">
<exception class="Doctrine\Common\Annotations\AnnotationException" message="[Semantical Error] The annotation "@Enum" in property Doctrine\ORM\Mapping\GeneratedValue::$strategy was never imported. Did you maybe forget to add a "use" statement for this annotation?">
<traces>
<trace>
at n/a in /home/user/www/claroline/claroline6/claro6-test/Claroline/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/AnnotationException.php line 54
</trace>
<trace>
at Doctrine\Common\Annotations\AnnotationException::semanticalError('The annotation "@Enum" in property Doctrine\ORM\Mapping\GeneratedValue::$strategy was never imported. Did you maybe forget to add a "use" statement for this annotation?') in /home/user/www/claroline/claroline6/claro6-test/Claroline/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/DocParser.php line 703
</trace>
<trace>
at Doctrine\Common\Annotations\DocParser->Annotation() in /home/user/www/claroline/claroline6/claro6-test/Claroline/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/DocParser.php line 641
</trace>
<trace>
at Doctrine\Common\Annotations\DocParser->Annotations() in /home/user/www/claroline/claroline6/claro6-test/Claroline/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/DocParser.php line 334
</trace>
<trace>
at Doctrine\Common\Annotations\DocParser->parse('/** * The type of Id generator. * * @var string * * @Enum({"AUTO", "SEQUENCE", "TABLE", "IDENTITY", "NONE", "UUID", "CUSTOM"}) */', 'property Doctrine\ORM\Mapping\GeneratedValue::$strategy') in /home/user/www/claroline/claroline6/claro6-test/Claroline/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/AnnotationReader.php line 226
</trace>
<trace>
at Doctrine\Common\Annotations\AnnotationReader->getPropertyAnnotations(object(ReflectionProperty)) in /home/user/www/claroline/claroline6/claro6-test/Claroline/app/cache/dev/classes.php line 7464
</trace>
<trace>
at Doctrine\Common\Annotations\FileCacheReader->getPropertyAnnotations(object(ReflectionProperty)) in /home/user/www/claroline/claroline6/claro6-test/Claroline/vendor/jms/serializer/src/JMS/Serializer/Metadata/Driver/AnnotationDriver.php line 140
</trace>
<trace>
at JMS\Serializer\Metadata\Driver\AnnotationDriver->loadMetadataForClass(object(ReflectionClass)) in /home/user/www/claroline/claroline6/claro6-test/Claroline/vendor/jms/metadata/src/Metadata/Driver/DriverChain.php line 38
</trace>
<trace>
at Metadata\Driver\DriverChain->loadMetadataForClass(object(ReflectionClass)) in /home/user/www/claroline/claroline6/claro6-test/Claroline/vendor/jms/serializer/src/JMS/Serializer/Metadata/Driver/AbstractDoctrineTypeDriver.php line 80
</trace>
<trace>
at JMS\Serializer\Metadata\Driver\AbstractDoctrineTypeDriver->loadMetadataForClass(object(ReflectionClass)) in /home/user/www/claroline/claroline6/claro6-test/Claroline/vendor/jms/metadata/src/Metadata/Driver/LazyLoadingDriver.php line 23
</trace>
<trace>
at Metadata\Driver\LazyLoadingDriver->loadMetadataForClass(object(ReflectionClass)) in /home/user/www/claroline/claroline6/claro6-test/Claroline/vendor/jms/metadata/src/Metadata/MetadataFactory.php line 103
</trace>
<trace>
at Metadata\MetadataFactory->getMetadataForClass('Doctrine\ORM\Mapping\GeneratedValue') in /home/user/www/claroline/claroline6/claro6-test/Claroline/vendor/jms/serializer/src/JMS/Serializer/GraphNavigator.php line 180
</trace>
<trace>
at JMS\Serializer\GraphNavigator->accept(object(GeneratedValue), null, object(SerializationContext)) in /home/user/www/claroline/claroline6/claro6-test/Claroline/vendor/jms/serializer/src/JMS/Serializer/XmlSerializationVisitor.php line 176
</trace>
<trace>
at JMS\Serializer\XmlSerializationVisitor->visitArray(array(object(Id), object(Column), object(GeneratedValue), object(Groups)), array('name' => 'array', 'params' => array()), object(SerializationContext)) in /home/user/www/claroline/claroline6/claro6-test/Claroline/vendor/jms/serializer/src/JMS/Serializer/GraphNavigator.php line 129
</trace>
```

**Comment by CPASimUSante** [comment]
This is here : (still not working in 6.3.8)

![notif](https://cloud.githubusercontent.com/assets/14015697/9981742/c995e360-5fc5-11e5-9e08-7a3604701d4a.png)

**Comment by ptsavdar** [comment]
Fixed in v6.0.1

**Comment by stmolivier** [comment]
thanks ! it's working now :)
